### PR TITLE
{chem}[intel/2015b] QuantumESPRESSO 5.2.1 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
requires https://github.com/hpcugent/easybuild-easyblocks/pull/714

Because the QE source URLs are sort of broken (don't yield a 404 on missing sources), the source files for QE need to be downloaded manually for now.

This shouldn't block this PR though. I made very sure that the right sources are being used by including checksums for all of them.

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review?

File: easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-5.2.1_yambo-fixes.patch [hunk 1]
```diff
@@ -0,0 +1,57 @@
+--- espresso-5.2.1/install/plugins_makefile.orig	2015-09-24 09:26:40.000000000 +0200
```
Comment (wpoely86) [inline easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-5.2.1_yambo-fixes.patch]: add a comment explaining the patch and claiming ownership

Comment (boegel) [inline easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-5.2.1_yambo-fixes.patch]: ah, I always forget this >_<

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/c09e72b888c0e0bd7fa5 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/d01329a63b088dc3c411 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/1ca1c3e60cf7d75ac74f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/a3c6085209e63e687a74 for a full test report.

**Comment by boegel** [comment]
Working as expected now, good to go with https://github.com/hpcugent/easybuild-easyblocks/pull/714 merged.

Thanks for the review @wpoely86!

**Pull request merged**